site/zenodo_rdm/migrator/extract.py

```diff
@@ -100,6 +100,25 @@ def run(self):
     ) TO STDOUT;
 """
 
+EXTRACT_DRAFTS_SQL = """
```

Comment:
this will be updated by alex

Reply:
No, the one here is correct, it's the same I have :)